config/web-app/istio.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: models-web-app
+spec:
+  gateways:
+  - knative-ingress-gateway.knative-serving
+  hosts:
+  - '*'
+  http:
+  - match:
+    - uri:
+        prefix: /models/
```

Comment:
Not a big deal, but I noticed when deploying standalone when visiting the app, if I didn't have the trailing slash, I'd get a 404. Is it okay to remove the trailing slash in the prefix?

Reply:
This makes sense, but it will be a little bit more involved than only changing the prefix in the VirtualService.

The [common code](https://github.com/kubeflow/kubeflow/blob/master/components/crud-web-apps/common/backend/kubeflow/kubeflow/crud_backend/helpers.py#L20) sets the `<base href="${PREFIX}/">` to the `index.html` of all the web apps right now, and that prefix always has a backslash at the end.

Making the apps work without a trailing backslash makes sense for all of Kubeflow's web apps so I'll open an issue in `kubeflow/kubeflow` to track this. Until I fix this we will need to keep the backslash here as well.

Does this sound OK to you?

Reply:
Sure, that's fine. Can leave it as is.

Reply:
Created an issue to track this effort in kubeflow/kubeflow https://github.com/kubeflow/kubeflow/issues/5802